tools/trap/ramdumpParser.py

```diff
@@ -1119,7 +1147,7 @@ def main():
 		SIZE_OF_ALLOC_NODE = 16
 
 		max_tasks = 0
-		# get MAX_TASKS num
+		# Get MAX_TASKS num
```

Comment:
You should not mix two different topics of changes in a single commit. Could you split this kind of change from operation change?